.gitignore

```diff
@@ -11,4 +11,6 @@ opengisch_rsa.pub
 *.pyc
 *.DS_Store
 .idea
-.env
\ No newline at end of file
+.env
+Pipfile
+Pipfile.lock
```

Comment:
https://stackoverflow.com/a/46303305/2319028

Reply:
```suggestion
# The pipfile will have the exact python version and breaks builds on other systems
Pipfile.lock
```

Reply:
Yes, I think having a requirements.txt file with versions should be enough for this project